frontend/packages/vkt/src/components/clerkEnrollment/overview/MoveModal.tsx

```diff
@@ -0,0 +1,160 @@
+import { FC, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  ComboBox,
+  CustomButton,
+  H3,
+  LoadingProgressIndicator,
+} from 'shared/components';
+import {
+  APIResponseStatus,
+  Color,
+  Severity,
+  TextFieldVariant,
+  Variant,
+} from 'shared/enums';
+import { useToast } from 'shared/hooks';
+import { AutocompleteValue, ComboBoxOption } from 'shared/interfaces';
+import { DateUtils } from 'shared/utils';
+
+import { useClerkTranslation, useCommonTranslation } from 'configs/i18n';
+import { useAppDispatch, useAppSelector } from 'configs/redux';
+import { AppRoutes } from 'enums/app';
+import { ClerkEnrollment } from 'interfaces/clerkExamEvent';
+import { ClerkListExamEvent } from 'interfaces/clerkListExamEvent';
+import {
+  moveEnrollment,
+  resetMoveEnrollment,
+} from 'redux/reducers/clerkEnrollmentDetails';
+import {
+  loadExamEvents,
+  resetClerkListExamEvent,
+} from 'redux/reducers/clerkListExamEvent';
+import { clerkEnrollmentDetailsSelector } from 'redux/selectors/clerkEnrollmentDetails';
+import { clerkExamEventOverviewSelector } from 'redux/selectors/clerkExamEventOverview';
+import { clerkListExamEventsSelector } from 'redux/selectors/clerkListExamEvent';
+import { ExamEventUtils } from 'utils/examEvent';
+
+interface MoveModalProps {
+  enrollment: ClerkEnrollment;
+  onCancel: () => void;
+}
+
+export const MoveModal: FC<MoveModalProps> = ({ enrollment, onCancel }) => {
+  const { t } = useClerkTranslation({
+    keyPrefix: 'vkt.component.clerkEnrollmentDetails.moveModal',
+  });
+  const translateCommon = useCommonTranslation();
+
+  const [selectedExamEventOption, setSelectedExamEventOption] =
+    useState<ComboBoxOption | null>(null);
+
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const { showToast } = useToast();
+
+  const { moveStatus } = useAppSelector(clerkEnrollmentDetailsSelector);
+  const { status: examEventListStatus, examEvents } = useAppSelector(
+    clerkListExamEventsSelector
+  );
+  const { examEvent } = useAppSelector(clerkExamEventOverviewSelector);
+
+  useEffect(() => {
+    if (examEventListStatus === APIResponseStatus.NotStarted) {
+      dispatch(loadExamEvents());
+    }
+  }, [dispatch, examEventListStatus]);
+
+  useEffect(() => {
+    if (moveStatus === APIResponseStatus.Success) {
+      showToast({
+        severity: Severity.Success,
+        description: t('successToast'),
+      });
+      dispatch(resetMoveEnrollment());
+      dispatch(resetClerkListExamEvent());
+      navigate(AppRoutes.ClerkHomePage);
```

Comment:
Tässä tapauksessa voisi toimia, että antaa navigate:n options objectille: `{ replace: true }` niin käyttäjä ei pysty enää navigoimaan takaisin vanhentuneeseen ilmoittautusnäkymään

Reply:
Tää ei taida itse asiassa lopulta muuttaa mitään. Jos tuo siis johtaa siihen, että selaimen takaisin-nappi viekin tällöin edeltävää sivua edeltäneeseen sivuun, avautuu tutkintotilaisuuden sivu oli tuo asetettuna tai ei. Kun tilaisuuden sivun url kuitenkin sama kuin ilmoittautumisen tarkastelusivun.

Reply:
Niin eli jos ei oo replace asetettuna:

1. Siirrä tutkinto
2. Mee historiassa kolme askelta taaksepäin (3x selain back)
3. Nyt avautuu äskettäin siirretyn henkilön tiedot vanhan tutkintotilaisuuden alta (url:ssa väärä id)

Jos replace asetettu, niin tuohon stale näkymään ei enää pääsee selaimen back:llä